jwm.md

```diff
@@ -94,4 +94,8 @@ For digital signatures the following algorithms MUST be supported.
 
 ## Signed and Encrypted
 
-When a sender would like for their message to feature both a non-repudiable digital signature and be encrypted to a recipient or multiple recipients. They can combine the encryption and signing operations defined earlier in this section by creating a [nested JWM](https://tools.ietf.org/html/draft-looker-jwm-01#section-1.2).
\ No newline at end of file
+When a sender would like for their message to feature both a non-repudiable digital signature and be encrypted to a recipient or multiple recipients. They can combine the encryption and signing operations defined earlier in this section by creating a [nested JWM](https://tools.ietf.org/html/draft-looker-jwm-01#section-1.2).
+
+## Message Authorization
+
```

Comment:
What does it mean for a message to be authorized?
Context for evaluation based on time, or version on the DID's Document?
[Notes from WG meeting]

Reply:
changed to authentication and added a note at the bottom to cover this.